prover/prover-kore

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+if [ $# -ne 1 ]; then
+  echo "Usage: $0 file.kore"
+  exit 1
+fi
+
+file="$1"
+
+./prover run --definition prover-kore "$file" -cCOMMANDLINE=.CommandLine -cPROVERDIR="\"`pwd`\""
```

Comment:
This is broken if run from another direwctory. Use add "prove_dir=$(cd $(dirname $0)/..; pwd -P)" to get the prover dir.
It is convention to do this at the first line of the file in case there are any change dirs later.

Reply:
The command you provided gives us the root of the repository, not the directory of the prover. Why is there `/..`?